geotorch/constraints.py

```diff
@@ -550,3 +550,28 @@ def positive_semidefinite_fixed_rank(
             a custom callable. Default: ``"expm"``
     """
     return _register_manifold(module, tensor_name, PSSDFixedRank, rank, f, triv)
+
+
+def alpha_stable(module: torch.nn.Module, tensor_name="weight", alpha=1e-3):
+    r"""Adds an alpha_stability parametrization to the matrix ``module.tensor_name``.
+
+    When accessing ``module.tensor_name``, the module will return the parametrized
+    version :math:`A` so that :math:`\min_{\lambda \in Sp(A)} \Re(\lambda)<= -\alpha`.
```

Comment:
Note that we don't use the spectrum notation here. Can you rewrite this following the notation we use in other manifolds where we discuss eigenvalues=

Reply:
nit. "Absolute value of the upper bound on the real part of the eigevenalues of :math: `X`", but otherwise LGTM